config/samples/resources/gkehubfeaturemembership/config-management-feature-membership/container_v1beta1_containercluster.yaml

```diff
@@ -17,7 +17,7 @@ kind: ContainerCluster
 metadata:
   annotations:
     cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-acm
-  name: gkehubfeaturemembership-dep-acm
```

Comment:
I think the previous naming is still correct for these dependent resources, right? Naming convention defined here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#dependency-resources

Reply:
Thanks for the pointer, revert the namings back